code/modules/vore/resizing/resize_vr.dm

```diff
@@ -45,6 +45,14 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
 	M.Translate(0, 16*(size_multiplier-1))
 	src.transform = M
 
+/mob/living/update_transform()
+	. = ..()
+	ASSERT(!ishuman(src))
```

Comment:
This proc already exists in `living.dm` on 1092, but it's commented out. You should probably just uncomment it and add a single line for size_multiplier scaling there rather than duplicating it all here.